Source/platform/ctr/system.cpp

```diff
@@ -9,11 +9,9 @@
 
 using namespace devilution;
 
-// Increase stack size for recursion in FindTransparencyValues()
-// 128 KB supports around 500 levels of recursion
-// Default stack size on 3DS is only 32 KB
+// Increased stack size to 128KB to allow higher levels of recursion and avoid game crashes

```

Comment:
What is the purpose of removing the extra details in the comment? Also, you have increased the stack size to 512 KB, not 128 KB.